Hunting Queries/BehaviorAnalytics/Anomalous Account Creation.yaml

```diff
@@ -15,7 +15,7 @@ query: |
 
  BehaviorAnalytics
  | where ActionType == "Add user"
- | where ActivityInsights.FirstTimeUserPerformedAction == True or ActivityInsights.FirstTimeActionPerformedInTenant == True
+ | where ActivityInsights contains "True"

```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done